spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -288,182 +313,121 @@ public interface PolledBarista {
-* <<spring-cloud-stream-overview-injecting-channels-directly>>
+In the preceding example, the created bound channel is named `inboundOrders`.
 
-[[spring-cloud-stream-overview-injecting-bound-interfaces]]
-===== Injecting Bound Interfaces
+While normally you don’t need to access individual channels and/or bindings directly (other then configuring them via `@EnableBinding` annotation), there may be
+times, such as testing or other corner cases, when you do.
 
-For each bound interface, Spring Cloud Stream generates a bean that implements the interface.
-Invoking an `@Input`-annotated or `@Output`-annotated method of one of these beans returns the relevant bound channel.
+Aside from generating channels for each binding and registering them as Spring beans, for each bound interface Spring Cloud Stream generates a bean that implements the interface.
+That means you can have access to the interfaces representing the bindings and/or individual channels by simply auto wiring either in your application (see two examples below)
 
-The bean in the following example sends a message on the output channel when its `hello` method is invoked and invokes `output()` on the injected `Source` bean to retrieve the target channel:
+_Autowire Binding interface_
 
-[source,java]
+[source, java]
 ----
-@Component
-public class SendingBean {
+@Autowire
+private Source source
 
-    private Source source;
-
-    @Autowired
-    public SendingBean(Source source) {
-        this.source = source;
-    }
-
-    public void sayHello(String name) {
-         source.output().send(MessageBuilder.withPayload(name).build());
-    }
+public void sayHello(String name) {
+    source.output().send(MessageBuilder.withPayload(name).build());
 }
 ----
 
-[[spring-cloud-stream-overview-injecting-channels-directly]]
-===== Injecting Channels Directly
-
-Bound channels can be also injected directly, as shown in the following example:
+_Autowire individual channel_
 
 [source, java]
 ----
-@Component
-public class SendingBean {
-
-    private MessageChannel output;
-
-    @Autowired
-    public SendingBean(MessageChannel output) {
-        this.output = output;
-    }
-
-    public void sayHello(String name) {
-         output.send(MessageBuilder.withPayload(name).build());
-    }
-}
-----
+@Autowire
+private MessageChannel output;
 
-If the name of the channel is customized on the declaring annotation, that name should be used instead of the method name.
-Consider the following declaration:
-
-[source,java]
-----
-public interface CustomSource {
-    ...
-    @Output("customOutput")
-    MessageChannel output();
+public void sayHello(String name) {
+    output.send(MessageBuilder.withPayload(name).build());
 }
 ----
 
-Given that declaration, the channel is injected as shown in the following example:
+You can also use standard Spring's `@Qualifier` annotation for cases when channel names are customized or in a multiple channesl scenarios specifically named channel is required
```

Comment:
Replace with "You can also use standard Spring's `@Qualifier` annotation for cases when channel names are customized or in multiple-channel scenarios that require specifically named channels." The following example shows how to use the `@Qualifier` annotation in this way: